qBittorrent/qBittorrent.php

```diff
@@ -30,7 +35,8 @@ public function login()
             'cookies' => $this->jar,
             'headers' => ['content-type' => 'application/x-www-form-urlencoded']
         ];
-        return parent::execute($this->url('login'), $attrs, false, 'POST');
+        #return parent::execute($this->url('login'), $attrs, false, 'POST');
+        return parent::execute($this->url('/api/v2/auth/login'), $attrs, false, 'POST');
```

Comment:
need to remove the slash at the front, this should fix the login issue

Reply:
Confirmed, thanks for that. Resolved in later commit.